monero/utils_test.go

```diff
@@ -1,45 +1,36 @@
 package monero
 
 import (
-	"sync"
+	"path"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/athanorlabs/atomic-swap/common"
 	mcrypto "github.com/athanorlabs/atomic-swap/crypto/monero"
-	"github.com/athanorlabs/atomic-swap/tests"
-
-	"github.com/stretchr/testify/require"
 )
 
 func TestWaitForBlocks(t *testing.T) {
-	c := NewWalletClient(tests.CreateWalletRPCService(t))
-	require.NoError(t, c.CreateWallet("wallet", ""))
-	daemon := NewDaemonClient(common.DefaultMoneroDaemonEndpoint)
-
-	addr, err := c.GetAddress(0)
-	require.NoError(t, err)
+	c := CreateWalletClient(t)
 
 	heightBefore, err := c.GetHeight()
 	require.NoError(t, err)
 
-	var wg sync.WaitGroup
-	wg.Add(1)
-	go func() {
-		errGen := daemon.GenerateBlocks(addr.Address, 181)
-		require.NoError(t, errGen)
-		wg.Done()
-	}()
```

Comment:
Background mining is automatically provided by `CreateWalletClient(t)`, so we no longer need these code blocks.